alvr/dashboard/src/data_sources.rs

```diff
@@ -231,7 +232,7 @@ impl DataSources {
                         match ws.read() {
                             Ok(tungstenite::Message::Text(json_string)) => {
                                 if let Ok(event) = serde_json::from_str(&json_string) {
-                                    debug!("Server event received: {:?}", event);
+                                    // debug!("Server event received: {:?}", event);
```

Comment:
Do we not want to keep this? Cleaning up the logging is a different issue

Reply:
Unintentional, it was spamming into logs too much, couldn't see anything else because of it, just commented for the dev time